src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+  const message = document.createElement('div');
+
+  message.className = 'message';
+
+  logo.addEventListener('click', () => {
+    const text = 'Promise was resolved!';
+
+    message.append(text);
+  });
```

Comment:
The event listener for the logo click resolves the promise, but it does not remove the event listener after the promise is resolved. Consider removing the event listener after the first click to prevent multiple resolutions.